src/Service/NullSearchService.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Algolia\SearchBundle\Service;
+
+use Algolia\AlgoliaSearch\RequestOptions\RequestOptions;
+use Algolia\AlgoliaSearch\Response\BatchIndexingResponse;
+use Algolia\AlgoliaSearch\Response\NullResponse;
+use Doctrine\Common\Persistence\ObjectManager;
+
+final class NullSearchService implements SearchServiceInterface
+{
+    /**
+     * @param string $className
+     *
+     * @return bool
+     */
+    public function isSearchable($className)
+    {
+        return true;
+    }
+
+    /**
+     * @return array<int, string>
+     */
+    public function getSearchables()
+    {
+        return ['foo', 'bar'];
+    }
+
+    /**
+     * @return array<string, array|int|string>
+     */
+    public function getConfiguration()
+    {
+        return ['batchSize' => 200];
+    }
+
+    /**
+     * @param string $className
+     *
+     * @return string
+     */
+    public function searchableAs($className)
+    {
+        return $className;
+    }
+
+    /**
+     * @param object|array<int, object>                      $searchables
+     * @param array<string, int|string|array>|RequestOptions $requestOptions
+     *
+     * @return \Algolia\AlgoliaSearch\Response\AbstractResponse
+     */
+    public function index(ObjectManager $objectManager, $searchables, $requestOptions = [])
+    {
+        return new BatchIndexingResponse(['success' => true], 'foo');
```

Comment:
This should return `new NullResponse();`.

Reply:
There is a reason why I specified `objectIds` in response of my implementation of nullclient. `SearchImportCommand` tries to access it at https://github.com/algolia/search-bundle/blob/7b02cead4e26549e4bb8f3f19efbb747be1fc925/src/Command/SearchImportCommand.php#L127

Reply:
@ostrolucky with a `NullResponse()` it doesn't enter the loop